runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1570,6 +1546,39 @@ TR_ResolvedJ9JITServerMethod::createResolvedMethodFromJ9MethodMirror(TR_Resolved
    packMethodInfo(methodInfo, resolvedMethod, fe);
    }
 
+bool
+TR_ResolvedJ9JITServerMethod::verifyCache(TR_ResolvedJ9JITServerMethodInfo clientMethodInfo, TR_ResolvedJ9JITServerMethodInfo cachedMethodInfo, TR_YesNoMaybe unresolvedInCache, bool *unresolvedInCP)
+   {
+   // 
+   if (std::get<0>(cachedMethodInfo).remoteMirror == NULL)
+      {
+      return true;
+      }
+   // If a method is interpreted, some of the parameters might differ from the cached one
+   // But since we are only caching for the duration of one compilation, we ignore the discrepancy
+   if (std::get<0>(clientMethodInfo).isInterpreted == std::get<0>(cachedMethodInfo).isInterpreted)
+      {
+      // cached method is interpreted nothing to compare
+      if (std::get<0>(cachedMethodInfo).isInterpreted)
+         {
+         return true;
+         }
+      if (unresolvedInCP)
+         {
+         if (unresolvedInCache == TR_maybe)
```

Comment:
Add a comment here saying that if  `unresolvedInCP` wasn't cached, then simply assume that updated info will be correct.